CONTRIBUTING.md

```diff
@@ -91,6 +92,8 @@ configured versions when changing into a directory.
 [semantic-release-badge]: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
 [commitizen-link]: http://commitizen.github.io/cz-cli/
 [commitizen-badge]: https://img.shields.io/badge/commitizen-friendly-brightgreen.svg
+[PR-tutorial]: https://github.com/firstcontributions/first-contributions
+[PR-course]: http://makeapullrequest.com
```

Comment:
rather than duplicating this link, can we just reuse one of the two variables in both cases?

Reply:
Makes sense. It should be updated now